File: April-Challenges-2021/Day-29/First-and-Last-Position-of-Elements-in-Sorted-Array.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        vector<int>ind;
        ind.push_back(-1);
        ind.push_back(-1);
        for(int i = 0; i<nums.size(); i++){
            if(nums[i]==target){
                ind[0] = i;
                break;
            }
        }
        if(ind[0]==-1){
            return ind;
        }
        for(int i = ind[0]; i<nums.size(); i++){
            if(nums[i]==target){
                ind[1] = i;
            }
        }
        return ind;
        
    }
};

File: April-Challenges-2021/Day-29/First-and-Last-Position-of-Elements-in-Sorted-Array.cs
public class Solution {
    public int[] SearchRange(int[] nums, int target) {
        int[] ind = new int[2];
        ind[0] = -1;
        ind[1] = -1;
        if(nums.Length==0){
            return ind;
        }
        for(int i = 0; i<nums.Length; i+=1){
            if(nums[i]==target){
                ind[0] = i;
                break;
            }
        }
        if(ind[0]==-1){
            return ind;
        }
        for(int i = ind[0]; i<nums.Length; i+=1){
            if(nums[i]==target){
                ind[1] = i;
            }
        }
        return ind;
    }
}
